### PR TITLE
Script to call goimports

### PR DESCRIPTION
**Pull request by justinsb**
Trying out python

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2140)

### CODE REVIEW

File: hack/verify-goimports [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# limitations under the License.
+
+import os
+from os import path
+import subprocess
+import sys
+
+from lib.kubernetes import devtools
+
+gopath=os.environ['GOPATH']
```
Comment (kris-nova) [inline hack/verify-goimports]: In _theory_ there could be a `$GOPATH` with multiple locations, but not a deal breaker at all. Just a thought.

Comment (kris-nova) [inline hack/verify-goimports]: In _theory_ there could be a `$GOPATH` with multiple locations, but not a deal breaker at all. Just a thought.

Comment (justinsb) [inline hack/verify-goimports]: Agreed, but it's a developer-only script for now and I wanted to start simple

Comment (kris-nova) [inline hack/verify-goimports]: Totes 👍🏻

**Pull request merged**